Remove labels.yaml

nikhita opened the pull request:
Update `labels.yaml` with latest labels.

Came across this while removing all "thirdpartyresource" occurrences.  Noticed that other labels also need update, so opened a new PR. :)

Edit: updated to remove the file entirely. 

**Release note**:
<!--  Write your release note:
1. Enter your extended release note in the below block. If the PR requires additional action from users switching to the new release, include the string "action required".
2. If no release note is required, just write "NONE".
-->
```release-note
NONE
```

nikhita commented on labels.yaml:
```diff
@@ -368,3 +386,5 @@ labels:
   color: ededed
 - name: triaged
   color: d455d0
+- name: ¯\_(ツ)_/¯
```
this is great! :smile:

xiangpengzhao replied:
I like this haha!
/shrug

nikhita commented:
/retest

nikhita commented:
Hmm, I think we can actually remove this file as it is not serving any purpose.

nikhita commented:
@BenTheElder Does sig-testing use this file somewhere?

BenTheElder commented:
I don't think so, the label syncer uses a config map in test-infra, not sure about the existence of any other tools. cc @fejta

fejta commented:
The current version of this file is https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.yaml

nikhita committed:
Remove labels.yaml

The canonical source is in the test-infra repo in
https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.yaml

nikhita commented:
/remove-lifecycle stale

nikhita commented:
> The current version of this file is https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.yaml

Removed the `labels.yaml` file from the root since the canonical source is in test-infra. PTAL.

nikhita commented:
/retest

nikhita commented:
@BenTheElder @fejta lgty?

BenTheElder reviewed (approved):
/lgtm

nikhita commented:
/retest

nikhita commented:
/assign @lavalamp 
for approval

nikhita commented:
/retest

On Sun 8 Jul, 2018, 2:04 PM k8s-ci-robot, <notifications@github.com> wrote:

> @nikhita <https://github.com/nikhita>: The following test *failed*, say
> /retest to rerun them all:
> Test name Commit Details Rerun command
> pull-kubernetes-e2e-kops-aws ba4a4f7
> <https://github.com/kubernetes/kubernetes/commit/ba4a4f73f91245dc2f4c78c8b01d2ce69bb99b7f>
> link
> <https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/55023/pull-kubernetes-e2e-kops-aws/96403/> /test
> pull-kubernetes-e2e-kops-aws
>
> Full PR test history <https://k8s-gubernator.appspot.com/pr/55023>. Your
> PR dashboard <https://k8s-gubernator.appspot.com/pr/nikhita>. Please help
> us cut down on flakes by linking to
> <https://git.k8s.io/community/contributors/devel/flaky-tests.md#filing-issues-for-flaky-tests>
> an open issue
> <https://github.com/kubernetes/kubernetes/issues?q=is:issue+is:open> when
> you hit one in your PR.
>
> Instructions for interacting with me using PR comments are available here
> <https://git.k8s.io/community/contributors/guide/pull-requests.md>. If
> you have questions or suggestions related to my behavior, please file an
> issue against the kubernetes/test-infra
> <https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:>
> repository. I understand the commands that are listed here
> <https://go.k8s.io/bot-commands>.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/55023#issuecomment-403272025>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APXA0MGy7X1k8ZeiaHpxhou7GdoQ2Ezaks5uEcQJgaJpZM4QQS-f>
> .
>

nikhita commented:
/assign @thockin 
for approval
